libraries/provider_repo.rb

```diff
@@ -27,6 +27,7 @@ class NetuitiveCookbook::NetuitiveRepoProvider < Chef::Provider::LWRPBase
       include_recipe 'apt'
       # add repo and key
       apt_repository 'netuitive' do
+        distribution ''
```

Comment:
I think the distribution will always need to be empty for apt based systems, because the repo paths are set up as: https://repos.app.netuitive.com/deb/dists/stable/main/. That is, the actual distribution involved doesn't matter, because there aren't any in the repo.